### PR TITLE
Editor: Update the Add button in the HTML Editor toolbar

### PR DESCRIPTION
**Pull request by Copons**
Follow up to #18719.

Combine "insert content" and "insert special" into one menu item in the HTML Editor toolbar.
All the options previously available in the "insert special" menu are now accessible from the one single button.

| Before | After |
| --- | --- |
|screen shot 2017-10-12 at 10 44 20 | screen shot 2017-10-12 at 10 32 21

### Testing

Compare the Add button in both Visual and HTML editor modes, and make sure they look and work the same.

### CODE REVIEW

File: client/post-editor/editor-html-toolbar/style.scss [hunk 1]
```diff
@@ -190,9 +185,9 @@
 			color: $gray;
 		}
 		span {
-			color: darken( $gray, 30% );
+			color: $gray-text-min;
```
Comment (iamtakashi) [inline client/post-editor/editor-html-toolbar/style.scss]: It's very minor, but to be consistent with other labels, I think the text colour here should be `darken( $gray, 20% );` which is `#4f748e`.

Comment (Copons) [reply client/post-editor/editor-html-toolbar/style.scss]: This is actually something I meant to point out and I forgot.
Recently [we talked](https://github.com/Automattic/wp-calypso/pull/18624#issuecomment-335421073) about text colors with @folletto and we settled on using `$gray-text-min` to simplify future tweaks.

In the end the difference with `darken( $gray, 20% )` isn't really visible (at least to my coder eyes 😄 ), so I'm not really opposed on using `darken` instead.

Comment (iamtakashi) [reply client/post-editor/editor-html-toolbar/style.scss]: I wouldn't have realised if I didn't see the diff :) Ignore me then, thanks for the info!

**Comment by iamtakashi** [comment]
Works as expected. Design wise 👍 Thanks for working on this, @Copons !!

**Comment by retrofox** [comment]
I wondered i18n could affect here:



Is this safe for all languages? I think it is.

**Comment by Copons** [comment]
@retrofox All those strings are correctly `translate()`d, so it shouldn't be a code concern.

**Comment by retrofox** [comment]
> @retrofox All those strings are correctly translate()d, so it shouldn't be a code concern.

Yes, I know Japoco. What I meant was the width of the text.

**Comment by Copons** [comment]
Oh I see, sorry!

The dropdown has a `min-width: 185px`, but no max set.
If you try to write a super long string via the dev tools, you'll see that it simply gets larger

**Comment by jhnstn** [comment]
@Copons thanks again for jumping on this !

**Pull request merged**